src/components/Dashboard/FilingHistoryList/FilingTemplate.vue

```diff
@@ -53,7 +53,10 @@
       </div>
     </v-expansion-panel-header>
 
-    <v-expansion-panel-content class="pt-5">
+    <v-expansion-panel-content
+      eager
```

Comment:
General note: we should probably use `eager` for all expansion panel content blocks to facilitate unit testing.

Reply:
Noted.